### ISSUE TITLE
Autoprefixer 7.1.0 gives no output for this when I use it via PostCSS

### ISSUE THREAD

**Issue by jitendravyas** [postcss/autoprefixer#930]
```
@keyframes introjs-hint-pulse-anim-inner {
  0% {
    transform: scale(0);
    opacity: 0.0;
  }
}
```

![monosnap 2017-10-25 12-38-28](https://user-images.githubusercontent.com/77309/31985216-f25025c8-b981-11e7-8002-efec1691c6ff.jpg)

But it works fine on https://autoprefixer.github.io/

![monosnap 2017-10-25 12-41-17](https://user-images.githubusercontent.com/77309/31985211-ebf3a0ce-b981-11e7-911c-6a0fed916964.jpg)

Also works fine with VScode autoprefixer plugin

I have this config

```
		postcss: {
			dev: {
				options: {
					syntax: require('postcss-scss'),
					processors: [require('autoprefixer')()]
				},

				src: 'app/styles/*.css'
			},
			dist: {
				options: {
					syntax: require('postcss-scss'),
					processors: [require('autoprefixer')()]
				},

				src: 'dist/public/styles/*.css'
			}
		},

```

Browserlists file

```
last 3 Chrome versions
last 3 Firefox versions
last 3 Edge versions
last 3 Opera versions
last 3 ChromeAndroid versions
last 3 Safari versions
```

**EDIT**

Updated Autoprefixer to latest 7.1.6 but same issue

**Edit 2**

I just checked that No vendor prefix for `transition: 0.5s;` too

**Edit 3**

basically, https://autoprefixer.github.io/ gives different output for `last 2 version` but PostCSS autoprefixer gives different output, for many properties for same list of browser

**Comment by ai** [comment]
Output depends on `caniuse-lite` version, because it contains Can I Use data to get browser versions.

**Comment by ai** [comment]
Also Autoprefixer *always* use PostCSS. autoprefixer.github.io or VS plugin just have PostCSS inside.

**Comment by jitendravyas** [comment]
I didn't get what should I do to solve this problem. Can you show me a direction?

**Comment by jitendravyas** [comment]
Should I make any change in my browser config?

**Comment by ai** [comment]
Run `npx browserslist "QUERIES"` you will get actual browsers version. Then check it with `transform` Can I Use. I think you just don’t need prefixes for `transform`.

**Comment by jitendravyas** [comment]
![monosnap 2017-10-25 23-11-11](https://user-images.githubusercontent.com/77309/32013822-dad93d1e-b9d9-11e7-9a8c-c3893e712ce7.png)

**Comment by ai** [comment]
Replace QUERIES to your queries 😉

**Comment by jitendravyas** [comment]
Same result

**Comment by jitendravyas** [comment]
I replace browserlist with browserlists now

![monosnap 2017-10-25 23-19-02](https://user-images.githubusercontent.com/77309/32014183-fb4a2198-b9da-11e7-873a-e5c2a3b480a9.png)

**Comment by jitendravyas** [comment]
And I use Yarn in the project not npm

**Comment by ai** [comment]
Replace QUERIES to your Browserdlist queries 😉 with list of browser queries that you use in your Browserslist.

**Comment by jitendravyas** [comment]
Ok I installed NPX and tried again and get this. List seems fine. What to do now?

![monosnap 2017-10-25 23-31-46](https://user-images.githubusercontent.com/77309/32014849-e4067700-b9dc-11e7-9dd8-ad6680084dd8.png)

**Comment by ai** [comment]
You was not need npx. You already had it.

Now you have exactly list of browsers versions. Open Can I Use and check does they need this prefixes.

**Comment by jitendravyas** [comment]
Can you help me to know, How to know which version needs prefix and which not http://caniuse.com/#search=transform

**Comment by ai** [comment]
Just continue to is Autoprefixer and don't worry.
